Automattic/vip-go-mu-plugins
# Cleans_stale_notice

## Description

Adds removing stale dismissed identifiers in every ~ 1 in 100 admin requests.

Also sneaked in the fix for adding multiple dismissibility keys.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Define few (at least 3) notices that are dismissible
2. Dismiss two of them
3 On refresh only see one remaining
